index.html

```diff
@@ -241,14 +241,11 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
 The web enables freedom of expression
   </h3>
   <p>
-We will create web technologies and platforms
-that encourage free expression,
-where that does not contravene other human rights.
+We will create web technologies and platforms that encourage free expression.
 Our work should not enable state censorship, <a data-cite="RFC7258#section-1">surveillance</a>
 or other practices that seek to limit this freedom.
 This principle must be balanced with respect for other human rights
 and does not imply that individual services on the web must therefore support all speech.
-(For example: hate speech, harassment or abuse may reasonably be denied a platform).
```

Comment:
Could we keep this change to a different PR? I think I oppose it, and I wouldn't want the discussion to delay getting the rest in.

Reply:
For the sake of replying to each comments, I agree that this should be another PR (and be discussed by the TF)